docs/data-sources/client.md

```diff
@@ -1,34 +1,207 @@
 ---
-layout: "auth0"
 page_title: "Data Source: auth0_client"
 description: |-
-Data source to retrieve a specific Auth0 Application client by 'client_id' or 'name'
+  Data source to retrieve a specific Auth0 Application client by 'client_id' or 'name'.
 ---
 
 # Data Source: auth0_client
 
-Data source to retrieve a specific Auth0 Application client by 'client_id' or 'name'
+Data source to retrieve a specific Auth0 Application client by 'client_id' or 'name'.
 
 ## Example Usage
 
-```hcl
+```terraform
+# An Auth0 Client loaded using its name.
 data "auth0_client" "some-client-by-name" {
   name = "Name of my Application"
 }
+
+# An Auth0 Client loaded using its ID.
 data "auth0_client" "some-client-by-id" {
   client_id = "abcdefghkijklmnopqrstuvwxyz0123456789"
 }
 ```
 
-## Argument Reference
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `client_id` (String) The ID of the client. If not provided, `name` must be set.
+- `name` (String) The name of the client. If not provided, `client_id` must be set.
+
+### Read-Only
+
+- `addons` (List of Object) Addons enabled for this client and their associated configurations (see [below for nested schema](#nestedatt--addons)).
+- `allowed_clients` (List of String) List of applications ID's that will be allowed to make delegation request. By default, all applications will be allowed.
+- `allowed_logout_urls` (List of String) URLs that Auth0 may redirect to after logout.
+- `allowed_origins` (List of String) URLs that represent valid origins for cross-origin resource sharing. By default, all your callback URLs will be allowed.
+- `app_type` (String) Type of application the client represents.
+- `callbacks` (List of String) URLs that Auth0 may call back to after a user authenticates for the client. Make sure to specify the protocol (https://) otherwise the callback may fail in some cases. With the exception of custom URI schemes for native clients, all callbacks should use protocol https://.
+- `client_metadata` (Map of String) Metadata associated with the client, in the form of an object with string values (max 255 chars). Maximum of 10 metadata properties allowed. Field names (max 255 chars) are alphanumeric and may only include the following special characters: `:,-+=_*?"/\()<>@ [Tab] [Space]`.
+- `client_secret` (String) Secret for the client. Keep this private. To access this attribute you need to add the `read:client_keys` scope to the Terraform client. Otherwise, the attribute will contain an empty string.
+- `cross_origin_auth` (Boolean) Whether this client can be used to make cross-origin authentication requests (true) or it is not allowed to make such requests (false).
+- `cross_origin_loc` (String) URL of the location in your site where the cross-origin verification takes place for the cross-origin auth flow when performing Auth in your own domain instead of Auth0 hosted login page.
```

Comment:
```suggestion
- `cross_origin_loc` (String) URL of the location in your site where the cross-origin verification takes place for the cross-origin auth flow when performing authentication in your own domain instead of Auth0 Universal Login page.
```